lib/streamlit/type_util.py

```diff
@@ -227,6 +232,18 @@ def get_fqn_type(obj: object) -> str:
     _NUMPY_ARRAY_TYPE_STR,
 )
 
+# We show a special "UnevaluatedDataFrame" warning for cached funcs
```

Comment:
where do we show this? It sounds like the comment might be better put into whatever utils-function uses these types

Reply:
Added more comments on this to the cache utils